packages/wonder-blocks-clickable/src/util/is-client-side-url.js

```diff
@@ -2,12 +2,13 @@
 /**
  * Returns:
  * - false for hrefs staring with http://, https://, //.
- * - false for '#' and 'javascript:void(0);'
+ * - false for '#', 'javascript:...', 'mailto:...', 'tel:...', etc.
  * - true for all other values, e.g. /foo/bar
  */
 export const isClientSideUrl = (href: string): boolean => {
     return (
         !/^(https?:)?\/\//i.test(href) &&
-        !/^(#|javascript:void\(0\);?)$/.test(href)
+        !/^[a-zA-Z]+:/.test(href) &&
+        href !== "#"
```

Comment:
I think that these protocols may also be able to include hyphens, perhaps we can be more permissive than just a-zA-Z?

Reply:
Good call.  Looking at https://en.wikipedia.org/wiki/List_of_URI_schemes it looks like `.` can be included too.